### ISSUE TITLE
Option to get previous behavior with respect to hiding keyboard and entering ESC

### ISSUE THREAD

**Issue by rafaeln** [termux/termux-app#43]
Now termux by default behaves standardly, with the triangle retracting the keyboard instead of sending the escape key. I like the old behavior better, with the triangle sending the escape key and a touch on the screen hiding the keyboard. I've managed to get the triangle=esc behavior back by setting that up on termux.properties, and I am wondering if I can also set the single touch on the screen to hide the keyboard. Is there a way?

**Comment by Neo-Oli** [comment]
I don't think that was the old behavior. Termux sends the tab as mouse event to the terminal. This allows you to move the cursor in vim or switch tabs in tmux with taps on the screen. You can still hide the keyboard from the left drawer menu.

**Comment by rafaeln** [comment]
Tou are right, and so am I. The old behavior that I mentioned was only in applications that didn't accept mouse input, like bash.
